### PR TITLE
some fixes for recent segmentation update

### PR DESCRIPTION
**Pull request by bertsky**

### CODE REVIEW

File: ocrd_cis/ocropy/segment.py [hunk 1]
```diff
@@ -652,10 +658,44 @@ def _process_element(self, element, ignore, image, coords, element_id, file_id,
...
+                line_polygon = polygon_for_parent(line_polygon, element)
                 # annotate result:
                 element.add_TextLine(TextLineType(id=line_id, Coords=CoordsType(
                     points=points_from_polygon(line_polygon))))
 
+
+def polygon_for_parent(polygon, parent):
+    """Clip polygon to parent polygon range.
+    
+    (Should be moved to ocrd_utils.coordinates_for_segment.)
```
Comment (kba) [inline ocrd_cis/ocropy/segment.py:676]: :+1:

**Comment by bertsky** [comment]
@finkf can you please have a look at 5f7eab73e8d3e8c9d1a0d18e5d71a873e2626be9 and 9e02692229f257a5421375465d768a3a57fbdd68 and give me tumbs up (or a review) if you are okay with those changes to the README?

**Comment by finkf** [comment]
Looks good to me :+1:

**Comment by bertsky** [comment]
Oh wow, I completely forgot to merge. Thanks @finkf!

**Pull request merged**